### PR TITLE
Feature/update vrules

### PR DESCRIPTION
**Pull request by iariasleon**

### CODE REVIEW

File: test/acceptance/integration/Rules/Cards/card_create_rules.feature [hunk 1]
```diff
@@ -171,8 +171,8 @@ Feature: Append a new rule in Perseo manager using cards from portal
     | test_900003 | room_*      | updateAttribute  | DANGER                                                | ALARM           |
     | test_900004 | room.*      | SendSmsMibAction | ${device_latitude}${device_longitude}${measure.value} | 123456789       |
 
-  @not_updated_card_without_card
-  Scenario: append a new rule in Perseo manager using card from portal using only not updated card without action card
+  @not_updated_card_without_AC
```
Comment (fgalan) [inline test/acceptance/integration/Rules/Cards/card_create_rules.feature]: AC could be mislead with "Access Control"... I would suggest for using the expanded name (action_card)

File: test/acceptance/integration/Rules/Cards/card_read_rules.feature [hunk 2]
```diff
@@ -51,8 +51,8 @@ Feature: get rules in Perseo manager from portal
   Examples:
     | rule_name     | attribute_name | operator     | data_type | value | action           | response                                              | parameters      |
     | test_00000001 | temperature    | GREATER_THAN | Quantity  | 34    | SendEmailAction  | ${device_latitude}${device_longitude}${measure.value} | erwer@sdfsf.com |
-    | test_00000002 | temperature    | MINOR_THAN   | Quantity  | 34.56 | SendSmsMibAction | body to response                                      | 123456789       |
-    | test_00000003 | temperature    | EQUAL_TO     | Quantity  | -34   | updateAttribute  | DANGER                                                | ALARM           |
+  #  | test_00000002 | temperature    | MINOR_THAN   | Quantity  | 34.56 | SendSmsMibAction | body to response                                      | 123456789       |
```
Comment (fgalan) [inline test/acceptance/integration/Rules/Cards/card_read_rules.feature]: Why is this commented? A FIXME explaining the reason would be great.

File: test/acceptance/integration/Rules/Cards/card_create_rules.feature (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/integration/Rules/Cards/card_create_rules.feature]: Fixed in 8e59734

File: test/acceptance/integration/Rules/Cards/card_read_rules.feature (hunk 2 above)
Comment (iariasleon) [inline test/acceptance/integration/Rules/Cards/card_read_rules.feature]: Fixed in d3c6696
Reason: these comments are not necessary

**Comment by jcalderin** [comment]
LGTM

**Comment by fgalan** [comment]
LGTM

**Pull request merged**